__init__.py

```diff
@@ -0,0 +1,48 @@
+bl_info = {
+    "name": "Cesium Ion",
```

Comment:
`ion` is always lower-cased.  The only exception is code symbols for readability `IonClass`, `CesiumIonClass`, etc..